Formula/podman.rb

```diff
@@ -2,10 +2,12 @@ class Podman < Formula
   desc "Tool for managing OCI containers and pods"
   homepage "https://podman.io/"
   license "Apache-2.0"
+  revision 1
```

Comment:
If this only affects relocatability then we probably don't need to force everyone to reinstall this with a `brew upgrade`.
```suggestion
```

Reply:
Updated.